aiven_db_migrate/migrate/pgutils.py

```diff
@@ -4,13 +4,16 @@
 from typing import Any, Dict, Optional
 from urllib.parse import parse_qs, urlparse
 
+import os
 import psycopg2
 import re
 import select
 import time
 
+usr_dir = os.getenv('PGHOMEDIR', '/usr')
```

Comment:
pghomedir (pg_home_dir) is a different thing AFAIK, where pg **data** dir is located. I'd call this PGBINDIR or PG_BIN_DIR (matching the current naming from aiven-db-migrate)

Also, I think this global usr_bin is a difficult-to-maintain design choice. My take would be to add an option to the cmdline parser in pgmigrate, and propagate that option to find_pgbin_dir usr_dir param.

Reply:
how does it sound if I make a cmdline arg `--pgbindir` instead?